packages/@o3r/rules-engine/.eslintrc.js

```diff
@@ -15,6 +15,24 @@ module.exports = {
     ],
     'sourceType': 'module'
   },
+  'overrides': [
+    {
+      'files': [
+        'package.json'
+      ],
+      'plugins': [
+        '@nx',
+      ],
+      'rules': {
+        '@nx/dependency-checks': ['error', {
+          'buildTargets': ['build', 'build-builders', 'compile', 'test'],
+          'checkObsoleteDependencies': false,
+          'checkVersionMismatches': false,
+          'ignoredDependencies': ['@o3r/test-helpers', '@o3r/configuration', '@o3r/localization', '@o3r/testing']
```

Comment:
is that not fixable using Nx `implicitDependencies` ?

Reply:
I tried it and it manually and does not do the fix.  
Anyway, I would expect to propose an autofix if it would be the case.